modules/sdk-coin-sui/test/unit/transactionBuilder/transferBuilder.ts

```diff
@@ -85,5 +89,120 @@ describe('Sui Transfer Builder', () => {
       };
       should(() => builder.gasData(invalidGasPayment)).throwError('Invalid payment, invalid or missing version');
     });
+
+    it('should submit a transaction', async () => {
```

Comment:
should we comment this out so we don't submit a transaction every time the unit tests run?